Packages/com.unity.inputsystem/InputSystem/Plugins/WebGL/WebGLJoystick.cs

```diff
@@ -1,10 +1,8 @@
 #if UNITY_WEBGL || UNITY_EDITOR
 
-////TODO
-
 namespace UnityEngine.InputSystem.WebGL
 {
-    public class WebGLJoystick : Joystick
+    public class WebGLJoystick : InputDevice
```

Comment:
What's the reason for this one?

Reply:
See PR description.

Reply:
I specifically asked you about this on slack two minutes after you posted this? :)